html/style.css

```
@@ -110,149 +99,109 @@
     font-family: 'Poppins', sans-serif;
 }
 
+.character-btns {
+    position: absolute;
+    bottom: 25px;
+    left: 20px;
+    right: 20px;
+}
+
 .character-btn {
-    position: static;
     width: 100%;
     height: 5.0vh;
     top: 65%;
     transition: 0.2s;
     float: left;
     margin-top: 10px;
-    margin-right: 10px;
     cursor: pointer;
     overflow: visible !important;
 }
 
-.character-btn>p {
-    color: #ededed;
-    letter-spacing: .1vh;
-    text-align: center;
-    font-family: 'Poppins', sans-serif;
-    font-size: 2vh;
-    text-transform: uppercase;
-    line-height: 5vh;
-    font-weight: bold;
-}
-
 #play {
-    display: none;
-    background-color: #28a745;
+    background-color: rgb(47 134 67);
     border-radius: .3vh;
     transition: all 0.2s ease;
 }
 
-#play:hover {
-    background-color: #228a3a;
-}
-
 #delete {
-    display: none;
-    background-color: #dc143c;
+    background-color: rgb(144 41 62);
     border-radius: .3vh;
     transition: all 0.2s ease;
 }
 
-#delete:hover {
-    background-color: #eb3434;
+#play:hover, #delete:hover {
+    box-shadow: rgb(17 17 37 / 25%) 0px 30px 60px -12px inset, rgb(54 80 154 / 30%) 0px 18px 36px -18px;
 }
 
 .character-info {
-    display: none;
+    min-width: 22.5%;
+    max-width: 27.5%;
     position: absolute;
-    width: 20%;
-    height: 50%;
     right: 14%;
-    top: -40%;
-    border-radius: .8vh;
+    top: 20%;
     padding: 30px;
-    background-color: rgb(23 23 23 / 80%);
+    background-color: rgb(27 33 49 / 95%);
+    border-radius: 5px;
+    box-shadow: rgb(0 0 0 / 16%) 0px 3px 6px, rgb(0 0 0 / 23%) 0px 3px 6px;
     overflow: visible !important;
+    transition: 400ms;
 }
 
-#char-info-icon {
-    position: relative;
-    font-size: 1.5vh;
-    color: #ededed;
-    right: 20px;
-    overflow: visible !important;
-}
-
-#char-info-titel {
+.char-info-titel {
```

```suggestion
.char-info-header {
```
Not your fault that it has this name but it is an opportunity to improve it, just to make it more readable